_i18n/fa.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: بازی‌های دوبعدی و سه‌بعدی، پروژه‌های چندسکویی و حتی ایده‌های ایکس.آر (واقعیت گسترده) خود را توسعه دهید!
```

And I added RTL support in #1069. So it's coming. I suggest that you add (in advance) this line:

```suggestion
meta:
  description: بازی‌های دوبعدی و سه‌بعدی، پروژه‌های چندسکویی و حتی ایده‌های ایکس.آر (واقعیت گسترده) خود را توسعه دهید!
  dir: rtl
```

---

should i add it after each string?